tox.ini

```diff
@@ -4,9 +4,10 @@
 # and then run "tox" from this directory.
 
 [tox]
```

Comment:
I we are adding a `pyproject.toml`, then I would like tox settings to be moved there too.

Reply:
Sure! :)